scalar.proto.ts

```diff
@@ -4,6 +4,7 @@ export type CertificateRegistrationRequest = {
   setCertHolderId: (certHolderId: string) => void;
   setCertVersion: (certVersion: number) => void;
   setCertPem: (certPem: string) => void;
+  serializeBinary: () => Uint8Array;
```

Comment:
Can review this file quickly because it's just the type definitions for Protocol Buffer compiler-generated classes.